src/app/inventory/store/stats.ts

```diff
@@ -599,9 +599,9 @@ function totalStat(stats: DimStat[]): DimStat {
   return {
     investmentValue: total,
     statHash: TOTAL_STAT_HASH,
-    displayProperties: ({
+    displayProperties: {
       name: t('Stats.Total'),
-    } as any) as DestinyDisplayPropertiesDefinition,
+    } as DestinyDisplayPropertiesDefinition,
```

Comment:
Is this not dangerous as DestinyDisplayPropertiesDefinition has a bunch of other properties? They are mostly strings but there is one array.

For reference,
```
export interface DestinyDisplayPropertiesDefinition {
  readonly description: string;
  readonly name: string;
  /**
   * Note that "icon" is sometimes misleading, and should be interpreted in the
   * context of the entity. For instance, in Destiny 1 the
   * DestinyRecordBookDefinition's icon was a big picture of a book.
   *
   * But usually, it will be a small square image that you can use as... well, an
   * icon.
   *
   * They are currently represented as 96px x 96px images.
   */
  readonly icon: string;
  readonly iconSequences: DestinyIconSequenceDefinition[];
  /**
   * If this item has a high-res icon (at least for now, many things won't), then the
   * path to that icon will be here.
   */
  readonly highResIcon: string;
  readonly hasIcon: boolean;
}
```

Reply:
completely fair, this relies on pre-established downstream behavior. they all just happen to be unused or primitives whose methods we don't end up using

Reply:
Yeah I wish the definitions' types were more accurate about what's optional.

Reply:
i have a branch of b-a-t with this but WOW is it obnoxious

Reply:
Yeah I've tried a few times and it's too much